includes/AppLeave.php

```diff
@@ -94,7 +94,10 @@
 	// create application table if not exists
 	//
 	$qry = "CREATE TABLE IF NOT EXISTS $appTy ($AppId BIGINT UNSIGNED AUTO_INCREMENT NOT NULL, $AppDate VARCHAR(10) NOT NULL, ";
+	
+	// current server date
```

Comment:
__[code_sniffer]__
* Spaces must be used for alignment; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->